dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseRegistryUpdater.java

```diff
@@ -71,6 +71,13 @@ private void updateRegistries() {
             MetadataImporter.loadRegistry(base + "dublin-core-types.xml", true);
             MetadataImporter.loadRegistry(base + "dcterms-types.xml", true);
             MetadataImporter.loadRegistry(base + "local-types.xml", true);
+            MetadataImporter.loadRegistry(base + "relationship-formats.xml", true);
```

Comment:
Note #2116

Reply:
This will need to be adjusted if #2116 is merged